classes/PrestaShopLogger.php

```diff
@@ -196,6 +206,8 @@ protected function isPresent()
 					AND `error_code` = \'' . $this->error_code . '\'
 					AND `object_type` = \'' . $this->object_type . '\'
 					AND `object_id` = \'' . $this->object_id . '\'
+                    AND `id_shop` = \'' . $this->id_shop . '\'
```

Comment:
Don't mix tabs and spaces. could you refactor the sql query?

Reply:
Fixed !

Reply:
Fixed !